src/app/ApplicationTemplate.Shared.Views/Content/Authentication/LoginPage.xaml

```diff
@@ -62,39 +61,43 @@
 					<local:DataValidationView Model="{Binding Form}"
 											  PropertyName="Email"
 											  Margin="0,28,0,0">
+						<!--
+						ue:ControlExtensions.LostFocusCommand="{Binding Form.ValidateProperty}"
```

Comment:
Could you add Nventive.View to enable all features from the login page?

Reply:
Yes I can do that, but it was for another PBI. Your call.